tests/Cms/Blueprints/FileBlueprintTest.php

```diff
@@ -0,0 +1,95 @@
+<?php
+
+use Kirby\Cms\Blueprint;
+use Kirby\Cms\File;
+use Kirby\Cms\Page;
+use Kirby\TestCase;
+
+/**
+ * @coversDefaultClass \Kirby\Cms\FileBlueprint
+ */
+class FileBlueprintTest extends TestCase
+{
+	public static function acceptAttributeProvider()
+	{
+		return [
+			[
+				'wildcard', // case name
```

Comment:
If we pass it as array key in the returned provider array instead of as an argument, PHPUnit will automatically use it as case name for error messages.